content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,116 @@
+  should be assigned to.
+- :guilabel:`Salesperson`: if the opportunities should be assigned to a specific salesperson, select
+  them from the drop-down menu. If no selection is made in this field, the opportunities are
+  assigned based on the team's existing rules.
+- :guilabel:`Marked Fields`: use this field to alter how the form handles marked fields. The default
+  option is to treat marked fields as :guilabel:`Required`, which is the recommended setting.
+- :guilabel:`Mark Text`: choose how :guilabel:`Marked Fields` should be identified. The default
+  character is an asterisk (`*`).
+- :guilabel:`Labels Width`: use this field to alter the pixel width of the labels, if desired.
+- :guilabel:`On Success`: select how the webpage reacts after a customer successfully submits a
+  form. :guilabel:`Nothing` keeps the customer on the same screen, with the addition of a
+  confirmation message that the form was submitted successfully. :guilabel:`Redirect` sends the
+  customer to a new webpage, based on the address in the field below. :guilabel:`Show Message`
+  replaces the form with a preconfigured message that informs the customer someone should respond to
+  them as soon as possible.
+- :guilabel:`URL`: if :guilabel:`Redirect` is selected in the :guilabel:`On Success` field, enter
+  the URL for the webpage where customers should be directed after successfully submitting a form.
+- :guilabel:`Visibility`: use the drop-down menu to add any visibility conditions for this field, if
+  desired.
+
+.. image:: opportunities_form/form-submitted-successfully.png
+   :align: center
+   :alt: The screen displayed after a customer successfully submits a contact form.
+
+.. important::
+   If *leads* are activated in CRM settings, selecting :guilabel:`Create an Opportunity` generates a
+   lead instead. To activate leads, navigate to :menuselection:`CRM app --> Configuration -->
+   Settings` and tick the :guilabel:`Leads` checkbox, then click :guilabel:`Save`.
+
+Customize contact form fields
+-----------------------------
+
+In addition to the settings for the form, the settings for each field can be customized as well.
+With the web editor menu still open, click into a field to open the field configuration settings.
+The following options are available to customize a field:
+
+- :guilabel:`Type`: choose a custom field option or an existing field type.
+- :guilabel:`Input Type`: determine the type of information customers should input. Available
+  options are :guilabel:`Text`, :guilabel:`Email`, :guilabel:`Telephone`, or :guilabel:`Url`. The
+  selection made in this field limits the format that customers can use when entering information.
+- :guilabel:`Label`: enter the name for the field.
+- :guilabel:`Position`: choose the way the label is aligned with the rest of the form. The label can
+  be hidden, above the field, to the left of the field, or right adjusted and closer to the field.
```

Comment:
do you want to consider tossing some FA icons in here for the corresponding options? Like, eye-slash for hidden, etc.? Might be cool. Just a suggestion, though. Totally up to you 👍

Reply:
I looked into this, but only the eye is an fa icon (the three arrows are svgs, not icons). Only including the one seemed out of place, I'd rather leave them all out for now